### ISSUE TITLE
Console does not scroll down after pyplot graph produced by remote command after 4.5.3

### ISSUE THREAD

**Issue by kakig** [jupyter/qtconsole#448]
What happens in this issue seems to be the similar to what happens in #347, but only after a pyplot graph is displayed. Otherwise the scroll down works normally. I use neovim with [jupyter-vim](https://github.com/jupyter-vim/jupyter-vim) to send remote commands to the qtconsole.

![Peek 2020-10-20 15-35](https://user-images.githubusercontent.com/36053545/96629391-ebf03500-12e9-11eb-892b-0e61359f0538.gif)

Issue #351 might also be relevant.

Version 4.5.2 works. On 4.5.3 the behavior seems identical to what happens in #347. On 4.7.7, the scroll down only stops working after a plot.

**Comment by kakig** [comment]
Sorry for the duplicate of #386, closing.
